### PR TITLE
releasenotes 2019.2.1

### PR DESCRIPTION
**Pull request by frogunder**
### What does this PR do?

Release notes for 2019.2.1. (unreleased)

### CODE REVIEW

File: doc/topics/releases/2019.2.1.rst [hunk 1]
```diff
@@ -32,3 +32,6772 @@ the YAML renderer that now require the new Jinja filter `tojson`.
...
+==========
+
+- Total Merges: **536**
+- Total Issue References: **69**
+- Total PR References: **350**
+
+- Contributors: **49** (`Akm0d`_, `Ch3LL`_, `DmitryKuzmenko`_, `Ethyling`_, `FireGrace`_, `KChandrashekhar`_, `ScoreUnder`_, `amendlik`_, `aplanas`_, `arsiesys`_, `bbinet`_, `bryceml`_, `cbosdo`_, `cdalvaro`_, `chdamianos`_, `cmcmarrow`_, `cro`_, `damianosSemmle`_, `dmurphy18`_, `doesitblend`_, `dwoz`_, `felippeb`_, `frogunder`_, `garethgreenaway`_, `github-abcde`_, `isbm`_, `jfindlay`_, `lomeroe`_, `mattLLVW`_, `mattp-`_, `mirceaulinic`_, `nicholasmhughes`_, `rbthomp`_, `rombert`_, `rsmekala`_, `s0undt3ch`_, `sathieu`_, `sbrennan4`_, `sdodsley`_, `simonflood`_, `sjorge`_, `soer7022`_, `stratusjerry`_, `tanlingyun2005`_, `terminalmage`_, `twangboy`_, `waynew`_, `weswhet`_, `xuhcc`_)
+
+
+INSERT RELEASE NOTES BODY HERE
```
Comment (Ch3LL) [inline doc/topics/releases/2019.2.1.rst]: need to remove this line here
```suggestion
```

File: doc/topics/releases/2019.2.1.rst [hunk 2]
```diff
@@ -32,3 +32,6772 @@ the YAML renderer that now require the new Jinja filter `tojson`.
...
+
+
+Changelog for v2019.2.0..04e28cff9bf42885e0e658326dd1ff0864f7fb18
```
Comment (Ch3LL) [inline doc/topics/releases/2019.2.1.rst]:
```suggestion
Changelog for v2019.2.0..v2019.2.1
```

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: doc/topics/releases/2019.2.1.rst [hunk 3]
```diff
@@ -32,3 +32,6772 @@ the YAML renderer that now require the new Jinja filter `tojson`.
 In 2019.2.1, we introduce a new configuration option for both the Salt master and Salt minion
 configurations to be able to support the older YAML renderer.  Using the option
 `use_yamlloader_old` will allow the YAML renderer to function as before.
+Statistics
```
Comment (Ch3LL) [inline doc/topics/releases/2019.2.1.rst]:
```suggestion

Statistics
```

File: doc/topics/releases/2019.2.1.rst [hunk 4]
```diff
@@ -32,3 +32,6771 @@ the YAML renderer that now require the new Jinja filter `tojson`.
...
+==========
+
+- Total Merges: **536**
+- Total Issue References: **69**
+- Total PR References: **350**
+
+- Contributors: **49** (`Akm0d`_, `Ch3LL`_, `DmitryKuzmenko`_, `Ethyling`_, `FireGrace`_, `KChandrashekhar`_, `ScoreUnder`_, `amendlik`_, `aplanas`_, `arsiesys`_, `bbinet`_, `bryceml`_, `cbosdo`_, `cdalvaro`_, `chdamianos`_, `cmcmarrow`_, `cro`_, `damianosSemmle`_, `dmurphy18`_, `doesitblend`_, `dwoz`_, `felippeb`_, `frogunder`_, `garethgreenaway`_, `github-abcde`_, `isbm`_, `jfindlay`_, `lomeroe`_, `mattLLVW`_, `mattp-`_, `mirceaulinic`_, `nicholasmhughes`_, `rbthomp`_, `rombert`_, `rsmekala`_, `s0undt3ch`_, `sathieu`_, `sbrennan4`_, `sdodsley`_, `simonflood`_, `sjorge`_, `soer7022`_, `stratusjerry`_, `tanlingyun2005`_, `terminalmage`_, `twangboy`_, `waynew`_, `weswhet`_, `xuhcc`_)
+
+
```
Comment (Ch3LL) [inline doc/topics/releases/2019.2.1.rst]: probably can remove a couple lines here

**Review by Ch3LL — CHANGES_REQUESTED** [review]

File: doc/topics/releases/2019.2.1.rst [hunk 5]
```diff
@@ -32,3 +32,6772 @@ the YAML renderer that now require the new Jinja filter `tojson`.
...
+
+
+Changelog for v2019.2.0..v2019.2.1
+=================================================================
```
Comment (Ch3LL) [inline doc/topics/releases/2019.2.1.rst]:
```suggestion
==================================
```

the '====' line needs to be equal with the text above

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Review by Ch3LL — APPROVED** [review]

**Review by dwoz — APPROVED** [review]

**Review by twangboy — APPROVED** [review]

**Pull request merged**